Presentation.Web/app/models/api/organization/organization-unit-dto.ts

```diff
@@ -1,12 +1,15 @@
 ﻿module Kitos.Models.Api.Organization {
     export interface IOrganizationUnitDto {
-        id: number
-        name: string
-        ean: string
-        localId: number
-        parentId: number
-        organizationId: number
-        origin: OrganizationUnitOrigin
-        externalOriginUuid: string | null
+        id: number;
+        name: string;
+        ean: string;
+        localId: number;
+        parentId: number;
+        organizationId: number;
+        origin: OrganizationUnitOrigin;
+        externalOriginUuid: string | null;
+        organization?: IOrganizationDto;
+        children?: IOrganizationUnitDto[];
+        draggable?: boolean;
```

Comment:
I had to add the properties to the Dto so I access them/assign the draggable. All of the nodes are of this type.

Or should I create a separate model, and map everything from the start?